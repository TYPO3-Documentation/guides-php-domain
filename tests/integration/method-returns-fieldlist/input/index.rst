=============================
PHP methods with return value
=============================

..  php:method:: getDate()

    Get the date.

    :returns: Either false on failure, or the datetime object for method chaining.
