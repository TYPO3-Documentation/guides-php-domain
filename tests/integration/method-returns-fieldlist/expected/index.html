<!-- content start -->
    <div class="section" id="php-methods-with-return-value">
            <h1>PHP methods with return value</h1>
            <dl class="php method">
    <dt class="sig sig-object php" id="getdate">
<span class="sig-name descname"><span class="pre">getDate</span></span>
<span class="sig-paren">(</span>
<span class="sig-paren">)</span>
 <a class="headerlink" href="#getdate" title="Permalink to this definition">Â¶</a>
    </dt>
    <dd>

    <p>Get the date.</p>

                    <dl class="field-list simple">
                <dt class="field-even">Returns</dt>
                <dd class="field-even">

    <p>Either false on failure, or the datetime object for method chaining.</p>

                </dd>
            </dl>
        </dd>
</dl>
    </div>
<!-- content end -->
